xml/cap_admin_minibroker.xml

```diff
@@ -0,0 +1,291 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE chapter
+[
+  <!ENTITY % entities SYSTEM "entity-decl.ent">
+    %entities;
+]>
+<chapter version="5.0" xml:id="cha.cap.minibroker"
+  xmlns="http://docbook.org/ns/docbook"
+  xmlns:xi="http://www.w3.org/2001/XInclude"
+  xmlns:xlink="http://www.w3.org/1999/xlink">
+ <info>
+  <title>Minibroker</title>
+  <dm:docmanager xmlns:dm="urn:x-suse:ns:docmanager">
+   <dm:bugtracker></dm:bugtracker>
+   <dm:translation>yes</dm:translation>
+  </dm:docmanager>
+ </info>
+ <para>
+  <link xlink:href="https://github.com/SUSE/minibroker">Minibroker</link> is an
+  <link xlink:href="https://www.openservicebrokerapi.org/">OSBAPI compliant
+  broker</link> created by members of the
+  <link xlink:href="https://github.com/osbkit">Microsoft Azure team</link>. It
+  provides a simple method to provision service brokers on Kubernetes clusters.
+ </para>
```

Comment:
This looks like a good place for the warning that Minibroker is not yet production-ready, and add some language that it is a technology preview for testing, or some such. "Technology preview" makes it sound cooler.

Reply:
Warning moved to this spot and message updated in https://github.com/SUSE/doc-cap/pull/285/commits/61faf662457f2be81a9b0a10e7df009ceca79c68